core/domain/question_fetchers_test.py

```diff
@@ -26,85 +26,102 @@
 
+from typing import Optional
+
+MYPY = False
+if MYPY:
+    from mypy_imports import question_models
+
 (question_models,) = models.Registry.import_models([models.NAMES.question])
 
 
 class QuestionFetchersUnitTests(test_utils.GenericTestBase):
     """Tests for question fetchers."""
 
-    def setUp(self):
+    def setUp(self) -> None:
         super(QuestionFetchersUnitTests, self).setUp()
         self.signup(self.EDITOR_EMAIL, self.EDITOR_USERNAME)
         self.signup(self.CURRICULUM_ADMIN_EMAIL, self.CURRICULUM_ADMIN_USERNAME)
 
-        self.admin_id = self.get_user_id_from_email(self.CURRICULUM_ADMIN_EMAIL)
-        self.editor_id = self.get_user_id_from_email(
+        self.admin_id = self.get_user_id_from_email(self.CURRICULUM_ADMIN_EMAIL) # type: ignore[no-untyped-call]
+        self.editor_id = self.get_user_id_from_email( # type: ignore[no-untyped-call]
             self.EDITOR_EMAIL)
 
-        self.set_curriculum_admins([self.CURRICULUM_ADMIN_USERNAME])
+        self.set_curriculum_admins([self.CURRICULUM_ADMIN_USERNAME]) # type: ignore[no-untyped-call]
 
-        self.admin = user_services.get_user_actions_info(self.admin_id)
-        self.editor = user_services.get_user_actions_info(self.editor_id)
+        self.admin = user_services.get_user_actions_info(self.admin_id) # type: ignore[no-untyped-call]
+        self.editor = user_services.get_user_actions_info(self.editor_id) # type: ignore[no-untyped-call]
 
-        self.save_new_skill(
+        self.save_new_skill( # type: ignore[no-untyped-call]
             'skill_1', self.admin_id, description='Skill Description 1')
-        self.save_new_skill(
+        self.save_new_skill( # type: ignore[no-untyped-call]
             'skill_2', self.admin_id, description='Skill Description 2')
 
-        self.question_id = question_services.get_new_question_id()
-        self.question = self.save_new_question(
+        self.question_id = question_services.get_new_question_id() # type: ignore[no-untyped-call]
+        self.question = self.save_new_question( # type: ignore[no-untyped-call]
             self.question_id, self.editor_id,
-            self._create_valid_question_data('ABC'), ['skill_1'])
+            self._create_valid_question_data('ABC'), ['skill_1']) # type: ignore[no-untyped-call]
 
-    def test_get_questions_and_skill_descriptions_by_skill_ids(self):
-        question_services.create_new_question_skill_link(
+    def test_get_questions_and_skill_descriptions_by_skill_ids(self) -> None:
+        question_services.create_new_question_skill_link( # type: ignore[no-untyped-call]
             self.editor_id, self.question_id, 'skill_1', 0.3)
 
         questions, _ = (
             question_fetchers.get_questions_and_skill_descriptions_by_skill_ids(
                 2, ['skill_1'], 0))
 
         self.assertEqual(len(questions), 1)
+        # Ruling out the possibility of None for mypy type checking.
+        assert isinstance(questions[0], question_domain.Question)
```

Comment:
```suggestion
        assert questions[0] is not None
```

Reply:
I think it's best to assert that it's not `None`.
Ditto elsewhere assertion is used for ruling out the possibility of None for mypy type checking.

Reply:
Done